projects/fal/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "fal"
-version = "0.12.6a0"
+version = "0.12.7a0"
```

Comment:
We seem to have enough changes for a minor release instead of a patch one. Not a biggie though esp since these are all alphas right now and there are no promises, but will need to settle that later.